torchmetrics/metric.py

```diff
@@ -256,7 +257,10 @@ def reset(self):
         """
         This method automatically resets the metric state variables to their default value.
         """
-        self._computed = None
+        # lower lightning versions requires this implicitly to log metric objects correctly
+        # in self.log
+        if _LIGHTNING_GREATER_THAN_1_2_8:
+            self._computed = None
```

Comment:
was this already fixed in PL 1.2.8?
at least my reading of this that there is a change in 1.2.8 and from this version will be used...